src/server.test.js

```diff
@@ -13,6 +13,40 @@ describe("Server", () => {
     app.server.close();
   });
 
+  describe("- live requests -", () => {
```

Comment:
I would suggest moving integration tests into a separate file for visibility, the idea is that they do not test a particular file in the code but the whole project end to end.

Let's also add one test that checks for specific error handling like malformed skylink.